hypervisor/include/arch/x86/board.h

```diff
@@ -28,8 +28,13 @@ struct vmsix_on_msi_info {
 extern struct dmar_info plat_dmar_info;
 
 #ifdef CONFIG_RDT_ENABLED
-extern struct platform_clos_info platform_l2_clos_array[MAX_PLATFORM_CLOS_NUM];
-extern struct platform_clos_info platform_l3_clos_array[MAX_PLATFORM_CLOS_NUM];
+#ifndef CONFIG_CDP_ENABLED
+#define MAX_CACHE_CLOS_NUM_ENTRIES MAX_PLATFORM_CLOS_NUM
```

Comment:
Along with MAX_PLATFORM_CLOS_NUM, can we define MAX_CACHE_CLOS_NUM_ENTRIES, MAX_MBA_CLOS_NUM_ENTRIES and in misc_cfg.h?